dvc/repo/experiments/save.py

```diff
@@ -11,6 +13,29 @@
 logger = logging.getLogger(__name__)
 
 
+def _save_experiment(
+    repo: "Repo",
+    baseline_rev: str,
+    force: bool,
+    name: Optional[str],
+    include_untracked: Optional[List[str]],
+) -> str:
+    stages = repo.commit([], force=force)
```

Comment:
```suggestion
    stages = repo.commit([], force=True)
```

I think this should be enough to replace #8691?

Reply:
It can still apply in rare cases when you do `dvc exp save -n some-existing-name`, right?

Reply:
> It can still apply in rare cases when you do dvc exp save -n some-existing-name, right?

True, btw @dtrifiro related to the above, the P.R. seems to lack a check for conflicts.